app/views/foods/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="is-flex is-justify-content-end mb-4">
+  <%= link_to t('add_food'), new_food_path, class: "button is-small add-food" %>
+</div>
+<div class="table-container">
+  <table class="table is-fullwidth">
+    <thead>
+      <tr>
+        <th>Food</th>
+        <th><abbr title="Measurement unit">Measurement</abbr></th>
+        <th><abbr title="Unit price">Price</abbr></th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @foods.each do |food| %>
+        <tr>
+          <th><%= food.name %></th>
+          <td><%= food.measurement_unit %></td>
+          <td>$<%= food.price %></td>
+          <td><%= link_to 'Delete', food, method: :delete, data: { confirm: 'Are you sure?' }, class: "button is-small delete-food" %></td>
+        </tr>
```

Comment:
[Optional]
- [ ] It will be really great if you use `partials` for rendering a collection of elements.